### PR TITLE
Proposal Syntax Sugar

### PR DESCRIPTION
**Pull request by joao-alberto**
#2 

I think which this is the simplest way to solve the proposal maintaining the old API without breaking changes.

### CODE REVIEW

File: src/grid/Col.js [hunk 1]
```diff
@@ -45,6 +46,7 @@ Col.propTypes = {
   medium: PropTypes.number,
   large: PropTypes.number,
   cols: PropTypes.number,
+  sizes: PropTypes.array,
```
Comment (sibelius) [inline src/grid/Col.js]: `PropTypes.arrayOf(PropTypes.number)`

**Comment by felippepuhle** [comment]
Great work! Could you add it do README.md?

**Comment by joao-alberto** [comment]
@felippepuhle Done 🤘

**Comment by felippepuhle** [comment]
Very nice! Thanks @psicotropicos 😄

**Pull request merged**